src/Modules/SimplCommerce.Module.Catalog/Views/Product/ProductOverview.cshtml

```diff
@@ -0,0 +1,233 @@
+@model SimplCommerce.Module.Catalog.ViewModels.ProductDetail
+@using SimplCommerce.Module.Core.Models
+
+
+@inject SignInManager<User> SignInManager
+
+@{
+    ViewBag.Title = Model.MetaTitle ?? Model.Name;
+}
+
+@*@section additionalMeta {
+    <meta name="title" content="@ViewBag.Title" />
+    <meta name="keywords" content="@Model.MetaKeywords" />
+    <meta name="description" content="@Model.MetaDescription" />
+}*@
+
+@*@section head
+{
+    <link simpl-append-version="true" rel="stylesheet" href="~/lib/smoothproducts/css/smoothproducts.css" />
+    <link simpl-append-version="true" rel="stylesheet" href="~/modules/catalog/product-detail.css" />
+}*@
```

Comment:
We cannot do this in a partial view. We don't need the meta tag for this circumstance. The css file can be included in the _Layout

Reply:
yes  I could remove it from ProductOverview.cshtml